data-prepper-plugins/otel-trace-raw-prepper/src/test/java/com/amazon/dataprepper/plugins/prepper/oteltrace/OTelTraceRawPrepperTest.java

```diff
@@ -220,36 +193,65 @@ public void testPrepareForShutdown() throws Exception {
         assertTrue(oTelTraceRawPrepper.isReadyForShutdown());
     }
 
-    private ExportTraceServiceRequest buildExportTraceServiceRequestFromJsonFile(String requestJsonFileName) throws IOException {
-        final StringBuilder jsonBuilder = new StringBuilder();
+    private static Span buildSpanFromJsonFile(final String jsonFileName) {
```

Comment:
We originally did this to easily support the ExportTraceServiceRequests. Do we need to build the spans from a JSON file too? Can we build the Span objects we want to use for testing?

Reply:
Yes, it is just a test utility method. The intention is to keep the test data in files as much as possible instead of hardcode all the test values inline.

Reply:
Thanks for the clarification. ExportTraceServiceRequests were larger objects when compared to our internal model. Building the test object from a JSON file made sense because the ETSR objects were so large. I am fine with keeping the files as long as we understand what we are optimizing for and we still consider our Spans to be large enough as well.